diff --git a/src/dev.zig b/src/dev.zig
index 1deb3260..41cc7879 100644
--- a/src/dev.zig
+++ b/src/dev.zig
@@ -197,12 +197,15 @@ pub const Env = enum {
                 else => Env.sema.supports(feature),
             },
             .wasm => switch (feature) {
-                .stdio_listen,
-                .incremental,
                 .wasm_backend,
                 .wasm_linker,
+                .legalize,
+                .build_exe_command,
+                .build_obj_command,
+                .sema,
+                .ast_gen,
                 => true,
-                else => Env.sema.supports(feature),
+                else => false,
             },
             .@"x86_64-linux" => switch (feature) {
                 .build_command,
